man/signal-cli-dbus.5.adoc

```diff
@@ -29,126 +29,252 @@ method(arg1<type>, arg2<type>, ...) -> return<type>
-* groupId : Byte array representing the internal group identifier
+Exception: Failure, InvalidNumber
 
-Note that quitting a group will not remove the group from the getGroupIds command, but set it inactive which can be tested with isMember()
+verifyWithPin(number<s>, verificationCode<s>, pin<s>) -> <>::
+* number            : Phone number
+* verificationCode  : Code received from Signal after successful registration request
+* pin               : PIN you set with setPin command after verifying previous registration
 
-Exceptions: GroupNotFound, Failure
+Exception: Failure, InvalidNumber
 
-isMember(groupId<ay>) -> active<b>::
-* groupId : Byte array representing the internal group identifier
+version() -> version<s>::
+* version : Version string of signal-cli
 
-Note that this method does not raise an Exception for a non-existing/unknown group but will simply return 0 (false)
+Exceptions: None
 
-sendEndSessionMessage(recipients<as>) -> <>::
-* recipients : Array of phone numbers 
+=== Methods for groups
 
-Exceptions: Failure, InvalidNumber, UntrustedIdentity
+getGroupIds() -> groupList<aay>::
+groupList : Array of Byte arrays representing the internal group identifiers
 
-sendGroupMessage(message<s>, attachments<as>, groupId<ay>) -> timestamp<x>::
-* message     : Text to send (can be UTF8)
-* attachments : String array of filenames to send as attachments (passed as filename, so need to be readable by the user signal-cli is running under)
-* groupId     : Byte array representing the internal group identifier
-* timestamp   : Can be used to identify the corresponding signal reply
+All groups known are returned, regardless of their active or blocked status. To query that use isMember() and isGroupBlocked()
 
-Exceptions: GroupNotFound, Failure, AttachmentInvalid
+Exceptions: None
 
-sendNoteToSelfMessage(message<s>, attachments<as>) -> timestamp<x>::
-* message     : Text to send (can be UTF8)
-* attachments : String array of filenames to send as attachments (passed as filename, so need to be readable by the user signal-cli is running under)
-* timestamp   : Can be used to identify the corresponding signal reply
+getGroupIds(dummy<s>) -> groupList<as>::
+* dummy     : any string (ignored by method; forces output to be identical with getGroupIdsBase64)
+* groupList : Array of strings representing the internal group identifiers
 
-Exceptions: Failure, AttachmentInvalid
+Exceptions: None
 
-sendMessage(message<s>, attachments<as>, recipient<s>) -> timestamp<x>::
-sendMessage(message<s>, attachments<as>, recipients<as>) -> timestamp<x>::
-* message     : Text to send (can be UTF8)
-* attachments : String array of filenames to send as attachments (passed as filename, so need to be readable by the user signal-cli is running under)
-* recipient   : Phone number of a single recipient
-* recipients  : Array of phone numbers 
-* timestamp   : Can be used to identify the corresponding signal reply
+getGroupIdsBase64() -> groupList<as>::
```

Comment:
Please remove the docs for methods from the other PRs here.

Reply:
done